### ISSUE TITLE
copyfighter doesnt seem to honour internal/vendor imports.

### ISSUE THREAD

**Issue by nnathan** [jmhodges/copyfighter#11]
copyfighter cannot find imports in internal/vendor directory. I'm not sure if this is a problem with copyfighter or the go/types being unable to walk the imports.

I currently get the error like:
`unable to type check package "main": scratch.go:5:2: could not import scratch/internal/foo (can't find import: scratch/internal/foo)`

Example program/structure below:

```
$ tree
.
├── internal
│   └── foo
│       └── foo.go
├── scratch
└── scratch.go
```

```
$ cat internal/foo/foo.go
package foo

type Bar struct {
    Ints [64]int
}
```

```
$ cat scratch.go
package main

import (
    "fmt"
    "scratch/internal/foo"
)

func printBar(b foo.Bar) {
    fmt.Println(b)
}

func main() {
    b := foo.Bar{}
    b.Ints[0] = 666
    printBar(b)
}
```

**Comment by zetsub0u** [comment]
Same here!

**Comment by sni** [comment]
any news on this?

**Comment by sni** [comment]
should be fixed with #13

**Comment by jmhodges** [comment]
Merged!
